psi4/src/psi4/libfock/v.h

```diff
@@ -170,8 +172,10 @@ class RV : public VBase {
     void initialize() override;
     void finalize() override;
 
+    // compute_V assuming same orbitals for different spin. Computes V_alpha, not spin-summed V.
     void compute_V(std::vector<SharedMatrix> ret) override;
-    void compute_Vx(std::vector<SharedMatrix> Dx, std::vector<SharedMatrix> ret) override;
+    /// compute_Vx assuming same orbitals for different spin. Vectors are length one.
```

Comment:
I think it can take multiple `Dx` as long as `Dx` and `ret` are the same size. Or i misunderstand what you mean with "vectors are length one".

Similar for the UV class.

Reply:
Edited comment, thanks. The crux is that these arguments come in (alpha, beta) pairs for UKS, but not for RKS.